x-pack/plugins/fleet/common/services/validate_package_policy.ts

```diff
@@ -198,6 +201,20 @@ export const validatePackagePolicyConfig = (
     parsedValue = value.trim();
   }
 
+  if (varDef === undefined) {
+    // eslint-disable-next-line no-console
+    console.debug(
+      i18n.translate('xpack.fleet.packagePolicyValidation.noVarDefErrorMessage', {
```

Comment:
debugging messages that aren't surfaced in UI probably don't need translation effort (at least we haven't done that thus far :))